### PR TITLE
Show risk field component instead of horizontal field.

### PR DESCRIPTION
**Pull request by elcorbs**
WHAT
Linked up the component for risk field so that when the backend has a risk: true marker this component will be shown. 
screen shot 2018-09-26 at 15 01 20

### CODE REVIEW

File: src/Components/ParentForm/ParentForm.test.js [hunk 1]
```diff
@@ -520,4 +520,42 @@ describe("<ParentForm>", () => {
...
+                    type: "string"
+                  }
+                }
+              }
+            }
+          }}
+          uiSchema={{
+            cat: {
+              "ui:field": "risk"
+
```
Comment (robinlacey) [inline src/Components/ParentForm/ParentForm.test.js]: 👀

File: src/Components/ParentForm/ParentForm.test.js [hunk 2]
```diff
@@ -520,4 +520,42 @@ describe("<ParentForm>", () => {
...
+            }
+          }}
+        />
+        );
+      expect(parentForm.find("RiskField").length).toEqual(1)
+
```
Comment (robinlacey) [inline src/Components/ParentForm/ParentForm.test.js]: remove empty line

File: src/Components/RiskField/index.js [hunk 3]
```diff
@@ -13,9 +13,9 @@ export default class RiskField extends React.Component {
         this.props.formData.riskBaselineMitigationsInPlace,
       riskCurrentReturnLikelihood:
         this.props.formData.riskCurrentReturnLikelihood || "1",
-      riskChangeRisk: this.props.formData.riskChangeRisk || "No",
-      riskCurrentMitigationsInPlace:
-        this.props.formData.riskCurrentMitigationsInPlace 
+      riskAnyChange: this.props.formData.riskAnyChange || "No",
```
Comment (robinlacey) [inline src/Components/RiskField/index.js]: confusing name - `riskAnyChange` suggests bool for dirty state

File: src/Components/RiskField/index.js [hunk 4]
```diff
@@ -13,9 +13,9 @@ export default class RiskField extends React.Component {
...
+      riskCurrentReturnMitigationsInPlace:
+        this.props.formData.riskCurrentReturnMitigationsInPlace
```
Comment (robinlacey) [inline src/Components/RiskField/index.js]: assuming this can never be null? as you're not doing the `<blah> || "No"` logic

Comment (ccp92) [reply src/Components/RiskField/index.js]: The front end has only been set up to accept Yes or No via a selector.

**Pull request merged**